src/components/molecules/Chatbox/Chatbox.tsx

```diff
@@ -59,29 +74,68 @@ export const Chatbox: React.FC<ChatboxProps> = ({
     [messages, deleteMessage]
   );
 
+  const dropdownOptions = useMemo(
+    () =>
+      OPTIONS.map((option) => (
+        <Dropdown.Item
+          key={option.name}
+          onClick={() => setActiveOption(option)}
+        >
+          {option.name}
+          <FontAwesomeIcon icon={option.icon} />
+        </Dropdown.Item>
+      )),
+    []
+  );
+
+  const showPoll = useMemo(() => activeOption?.name !== OPTIONS[0].name, [
+    activeOption,
+  ]);
+
+  const handleSubmit = useCallback((data) => console.log(data), []);
+
+  const renderForms = useMemo(() => {
+    switch (activeOption?.name) {
+      case OPTIONS[0].name:
```

Comment:
and here you would use that to explicitly get those values, without usage of `FOO[magical number]`

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1343/commits/b1c8ac0ec3e6df3cc363c91f3e36cf92b3ccfb00